netty/src/test/java/io/grpc/transport/netty/NettyClientHandlerTest.java

```diff
@@ -184,14 +183,13 @@ public void cancelBufferedStreamShouldChangeClientStreamStatus() throws Exceptio
 
     assertTrue(createPromise.isSuccess());
     verify(stream).transportReportStatus(eq(Status.CANCELLED), eq(true),
-        any(Metadata.Trailers.class));
+            any(Metadata.Trailers.class));
```

Comment:
I've reverted the formatting.

Agreed that full-file formatting is rather evil and should be avoided.  I generally don't use it but must have in this case.